docs/extensions/custom-bootstrapping.md

```diff
@@ -1,89 +1,169 @@
 ---
 title: Custom Bootstrapping
 ---
-Creating your own Bootstrap
-=============================
+Customising Bootstrapping
+=========================
 
-Whilst Bolt is designed to be simple for anyone to install, its core
-functionality is also modular and easy to configure for anyone who is
-comfortable making a few modifications to their bootstrap code.
+Whilst Bolt is designed to be simple for anyone to install, its core 
+functionality is also modular and easy to configure for developers who are
+comfortable making a few modifications to how Bolt is bootstrapped.
 
-By default Bolt ships with its own bootstrap file which is responsible for
-setting up and running a Bolt application. Most of the advanced configuration
-options are best attained by taking over this responsibility in your own
-application. In the root of your project you will see an `index.php` file that
-should look like this:
+There are two ways your Bolt application is used, via web requests and via the
+command line with `nut`. Because there are two entry points, bootstrapping is
+done in a common `bootstrap.php` file. 
 
-```
-$app = require dirname(__FILE__) . '/../vendor/bolt/bolt/app/web.php';
-if ($app === false) {
-    return false;
-}
-$app->run();
+One of the important functions of the bootstrap file is to read a loading 
+configuration file in the root directory of your project. This configuration is
```

Comment:
Yeah, I blame … um … well, not the :koala: & not the :hamster: … who can we needlessly label with blame here? :wink: 

I'll fix soon